lochness/redcap/__init__.py

```diff
@@ -107,6 +108,13 @@ def initialize_metadata(Lochness: 'Lochness object',
     df = pd.DataFrame()
     # for each record in pulled information, extract subject ID and source IDs
     for item in data:
+        if multistudy:
+            site_two_letters_redcap_id = item[redcap_id_colname][:2]
+            site_two_letters_study = study_name.split('_')[1]
```

Comment:
For this PR to work, are we requiring users to have `_` in the study name?

Reply:
Yes. This may restrict some of the use for other projects apart from U24/U01, but at least for U24/U01, `_` is anyway will be in the study name for Pronet and Prescient.

Reply:
Linking PR #35